src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -1377,11 +1554,15 @@ private static Args ParseFromFile(string fileName)
     private const uint DEFAULT_POH_PINNING_INTERVAL = 0;
     private const uint DEFAULT_POH_FINALIZABLE_INTERVAL = 0;
     private const uint DEFAULT_POH_SURV_INTERVAL = 0;
+    private const uint DEFAULT_REQ_POH_SURV_INTERVAL = 0;
 #endif
 
     private const uint DEFAULT_PINNING_INTERVAL = 100;
     private const uint DEFAULT_FINALIZABLE_INTERVAL = 0;
 
+    private const uint DEFAULT_REQ_SOH_SURV_INTERVAL = 3;
+    private const uint DEFAULT_REQ_LOH_SURV_INTERVAL = 2;
```

Comment:
this seems really tiny for LOH.

Reply:
Oh, you mean fewer objects in LOH should survive till the end of the request?